cardano-api/src/Cardano/Api/Eras.hs

```diff
@@ -87,3 +101,123 @@ pattern AsMary     = AsMaryEra
 {-# DEPRECATED AsAllegra "Use 'AsAllegraEra' instead" #-}
 {-# DEPRECATED AsMary    "Use 'AsMaryEra' instead" #-}
 
+-- ----------------------------------------------------------------------------
+-- Shelley-based eras
+--
+
+-- | While the Byron and Shelley eras are quite different, there are several
```

Comment:
This comment was repeated below on line 127

Reply:
well spotted thanks